azure-db/bin/create-db-user

```diff
@@ -30,6 +30,14 @@ PGSSLMODE=require
 #  /o\ get base user name (without server) for SQL commands
 NEW_USER_BASE=${NEW_USER%%@*} || $NEW_USER
 
+
+# check if DB already exist
+psql -abe "$PG_URL_ADMIN" -c "\c ${NEW_DB_NAME}"
+if [ $? -eq 0 ]; then
```

Comment:
The `psql` exits the script before the test can be made...

I dunno if we should 

```suggestion
psql -v ON_ERROR_STOP=1  -abe "$PG_URL_ADMIN" -c "\c ${NEW_DB_NAME}"
if [ $? -eq 0 ]; then
```

Or just 

```suggestion
if ! psql -abe "$PG_URL_ADMIN" -c "\c ${NEW_DB_NAME}"; then
```